src/external-server/docker-amarna.ts

```diff
@@ -0,0 +1,42 @@
+import { Image } from "@nomiclabs/hardhat-docker";
+import { DockerServer } from "./docker-server";
+import * as fs from "fs";
+
+export class AmarnaDocker extends DockerServer {
+    useShell = false;
+    /**
+     * @param image the Docker image to be used for running the container
+     * @param cairoPaths the paths specified in hardhat config cairoPaths
+     */
+    constructor(image: Image, private path: string) {
+        super(image, "127.0.0.1", null, "", "starknet-docker-proxy");
```

Comment:
It should probably use a different container name instead of `starknet-docker-proxy` since `StarknetDockerProxy` uses that one. Saying this to prevent name conflicts, although not sure if the two containers can be run at the same time.

Reply:
We use `-rm`, but should totally rename it!